.github/workflows/release-alpha.yml

```diff
@@ -4,25 +4,37 @@ on:
   workflow_dispatch:
```

Comment:
do we need a "branch" / "ref" input here?  otherwise I'm guessing it will run by default on `main`?

Reply:
the `ref` is always configurable for a workflow_dispatch, it will default to main but picking a different branch from the drop down will release from the chosen branch so we can alpha release unmerged features